.setup/ansible/playbooks/submitty_course_creation.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Submitty Course Creation
+  hosts: submitty
+  become: true
+
+  roles:
+    - role: submitty_term_creation
+      vars:
+        submitty_term_creation_key: term
+        submitty_term_creation_name: winter2024
+        submitty_term_creation_start_date: 01/01/2024
+        submitty_term_creation_end_date: 06/07/2024
+
+    - role: submitty_course_creation
```

Comment:
I haven't been to close to the discussions related to ansible but is this playbook used for setting up a developer instance or is the idea that universities will maintain their own playbooks to manage their submitty instance?

Reply:
I see these playbooks as examples of how a university could manage their infrastructure. Each university would set the parameters for their university in their own playbook. All universities would use the same roles with just different parameters. At least that is what I was thinking.

Reply:
The playbooks are used as an example, showing each University how they should set their parameters.